README.md

```diff
@@ -7,6 +7,7 @@
 <p align="center">
   <a href="https://github.com/ava-labs/hypersdk/actions/workflows/unit-tests.yml"><img src="https://github.com/ava-labs/hypersdk/actions/workflows/unit-tests.yml/badge.svg" /></a>
   <a href="https://github.com/ava-labs/hypersdk/actions/workflows/static-analysis.yml"><img src="https://github.com/ava-labs/hypersdk/actions/workflows/static-analysis.yml/badge.svg" /></a>
+  <a href="https://github.com/ava-labs/hypersdk/blob/main/LICENSE"><img src="https://img.shields.io/badge/License-BSD--3-blue" /></a>
```

Comment:
Looks like we can even use an auto-generated image that scans our repo for the correct license type, so if we ever change the license it auto-updates https://shields.io/category/license, (i.e https://img.shields.io/github/license/ava-labs/hypersdk for a default one)

Reply:
Yup good call -- I went there first but didn't see it. Updated the PR to use shields.io instead 👍